### PR TITLE
Mobile labs alerts banner

### PR DESCRIPTION
**Pull request by NathanielBennett**
## What does this change?

Mobile labs want a bespoke site message to upsell mobile alerts for the next round of Presidential Primaries. This will run on US election pages through Wednesday June 8, The promo shows if the user is visiting a US election page on Android chrome:
## What is the value of this and can you measure success?

The mobile labs guys wanna recruit 1000 people to use their test
## Does this affect other platforms - Amp, Apps, etc?

 No
## Screenshots

mobile-labs
## Request for comment

@gtrufitt

### CODE REVIEW

File: common/app/conf/switches/FeatureSwitches.scala [hunk 1]
```diff
@@ -379,4 +379,13 @@ trait FeatureSwitches {
     exposeClientSide = true
   )
 
+  //Owener Dotcom participation
+  val MobileLabPresidentialPrimaries = Switch(
+    SwitchGroup.Feature,
+    "mobile-lab-presidential-primaries",
+    "When ON, we present a promo banner to android, US edition users inviting them to subscribe to experimental primaries alerts",
+    safeState = Off,
+    sellByDate = new LocalDate(2016, 6, 7),
```
Comment (gtrufitt) [inline common/app/conf/switches/FeatureSwitches.scala]: I would run this until Wednesday as with the time difference people should be able to sign up until Tuesday (as the test is running on Tuesday)

File: static/src/javascripts/projects/common/modules/onward/mobile-labs-alerts.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+define([
+    'common/views/svgs',
+    'common/utils/config',
+    'common/utils/detect',
+    'common/utils/template',
+    'common/modules/ui/message',
+    '!text!common/views/mobile-labs/presidential-primary-alerts.html',
```
Comment (gtrufitt) [inline static/src/javascripts/projects/common/modules/onward/mobile-labs-alerts.js]: I don't think you need this first !, can be `text!common/` etc

**Comment by gtrufitt** [comment]
Once the switch date is updated LGTM 👍

File: static/src/stylesheets/module/_release-message.scss [hunk 3]
```diff
@@ -20,6 +20,23 @@
...
+    .site-message__close-btn {
+        top: $gs-baseline * 8;
+    }
+
+    .site-message__header-item-message {
+        @include fs-textSans(2);
+    }
+
+    .site-message__header-item-headline {
+        @include fs-header(2);
```
Comment (sammorrisdesign) [inline static/src/stylesheets/module/_release-message.scss]: I'd probably opt to use `fs-textSans(2)` and maybe put it in bold.

**Pull request merged**